docs/it/attributi_utente.rst

```diff
@@ -228,36 +200,42 @@ Tabella attributi utente
         }
 
      - |spid-icon| |cieid-icon|
-   * - **phone_number**
-     - extra anagrafica
+   * - **phone_number** |br| Categoria: extra anagrafica
      - Numero di telefono mobile. String. Stringa numerica senza spazi intermedi |br|
        Esempio: |br|
-       ``"phone_number": "VATIT-12345678901"``
+       ``"phone_number":"VATIT-12345678901"``
      - |spid-icon| |cieid-icon|
-   * - **email**
-     - extra anagrafica
+   * - **phone_number_verified** |br| Categoria: extra anagrafica
+     - Valore Booleano che indica se il numero di telefono mobile dell'utente è stato verificato dall'OP. 
+     - |cieid-icon|
+   * - **$PREFIX/landline_number** |br| Categoria: extra anagrafica
+     - Numero di telefono fisso. String. Stringa numerica senza spazi intermedi |br|
+       Esempio: |br|
+       ``"$PREFIX/landline_number":"VATIT-12345678901"``
+     - |cieid-icon|
+   * - **email** |br| Categoria: extra anagrafica
      - Indirizzo di posta elettronica. String. Formato standard indirizzo di posta elettronica |br|
        Esempio: |br|
-       ``"email": "name@domain.it"``
+       ``"email":"name@domain.it"``
      - |spid-icon| |cieid-icon|
-   * - **https://attributes.eid.gov.it/e_delivery_service**
-     - extra anagrafica
+   * - **email_verified** |br| Categoria: extra anagrafica
+     - Valore Booleano che indica se l'email dell'utente è stata verificata dall'OP. 
+     - |cieid-icon|
+   * - **$PREFIX/e_delivery_service** |br| Categoria: extra anagrafica
      - Domicilio digitale. Indirizzo casella PEC |br|
        Esempio: |br|
-       ``"https://attributes.eid.gov.it/e_delivery_service":"nome@pecdomain.it"``
-     - |cieid-icon|
-   * - **https://attributes.eid.gov.it/eid_exp_date**
-     - extra anagrafica
+       ``"$PREFIX/e_delivery_service":"nome@pecdomain.it"``
+     - |spid-icon| |cieid-icon|
+   * - **$PREFIX/eid_exp_date** |br| Categoria: extra anagrafica
```

Comment:
$PREFIX/id_exp_date
@nunzionapoli please fix on https://www.agid.gov.it/sites/default/files/repository_files/tabella_attributi_v.1.3.pdf

Reply:
risolto